keyboards/dtisaac/dosa40rgb/dosa40rgb.h

```diff
@@ -18,6 +18,13 @@
 
 #include "quantum.h"
 
+enum keyboard_keycodes {
+    LED_EN = SAFE_RANGE, // Toggle LED
+    KEYMAP_SAFE_RANGE
+};
+#undef SAFE_RANGE
```

Comment:
Does this actually work? `SAFE_RANGE` is not a define, it's an entry in the quantum_keycodes enum.

Reply:
I'm not sure,  "**zvecr**" requested to add SAFE_RANGE
[zvecr requested changes 26 days ago](https://github.com/qmk/qmk_firmware/pull/14876#discussion_r735182015)